### ISSUE TITLE
Popover on first appearance flashes

### ISSUE THREAD

**Issue by pbackup12345** [juliankrispel/react-text-selection-popover#26]
The popover window on first apperance is first positioned somewhere to the left, and then after it is visible it is positioned properly over the text.
This creates a visible flash as the popover materializes on the left of the screen and then it is positioned correctly.
Try the codesandbox example after reload where it is also happening.
Suggestion:
On first appearance the popover should first be positioned then made visible.

**Comment by anuragsr** [comment]
One workaround I found was to apply `opacity: 0` to the child class of `<Popover>` and update it to 1 using a `setTimeout` in `componentDidUpdate` of the component where Popover is used:

```
  componentDidUpdate(){
    setTimeout(() => {
      $(".ctn-pop-outer").css({ opacity: 1 }) // .ctn-pop-outer being the child of <Popover>
    }, 10)
  }
```

**Comment by juliankrispel** [comment]
Yea this has been a bit of a pain.

PRS for this are welcome btw. Otherwise I'll get to it when I get to it 🤷‍♂

**Comment by sumukhah** [comment]
can somebody direct me on how to tackle this?

**Comment by jerry604** [comment]
Similar to the above, though instead of using a `setTimeout`, I used CSS animations with delay of 0 in the `className` prop passed to the `Popover`

```
@keyframes showPopover {
  from { opacity: 0 },
  to { opacity: 1 },
}
...
```

**Comment by juliankrispel** [comment]
This is finally fixed with `2.0`, which is a complete rewrite btw.
